line-bot-spring-boot/src/test/java/com/linecorp/bot/spring/boot/support/ReplyByReturnValueConsumerTest.java

```diff
@@ -69,7 +70,7 @@
     public void setUp() {
         target = targetFactory.createForEvent(EVENT);
         when(lineMessagingClient.replyMessage(any()))
-                .thenReturn(CompletableFuture.completedFuture(new BotApiResponse("success", null)));
+                .thenReturn(completedFuture(new BotApiResponse("", " success", null)));
     }
 
     // Publich methods test
```

Comment:
[NITS] Please fix typo in `Publich`

Reply:
:pray: Filed https://github.com/line/line-bot-sdk-java/pull/401